ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -35,7 +38,36 @@ def cleanup(cluster_name, cluster_id):
     run_cmd(f"{oc_bin} destroy cluster --dir {cleanup_path} --log-level=debug")
 
 
-def main():
+def get_clusters_to_delete(time_to_delete):
+    """
+    Get all cluster names that their EC2 instances running time is greater
+    than the specified time to delete
+
+    Args:
+         time_to_delete (int): The maximum time in seconds that is allowed
+            for clusters to continue running
+
+    Returns:
+        list: The cluster names (e.g ebenahar-cluster-gqtd4) to be provided to the
+            ci-cleanup script
+
+    """
+    conn = boto3.resource('ec2', region_name='us-east-2')
```

Comment:
I see region_name hardcoded here and not configurable. I would like to see this configurable.

Reply:
fixed